modules/pipeline/providers/cron/compensator/provider.go

```diff
@@ -57,6 +58,7 @@ type provider struct {
 
 	jsonStore jsonstore.JsonStore
 	dbClient  *dbclient.Client
+	client    *db.Client
```

Comment:
should be merged into one dbClient

Reply:
At present, it is impossible to merge into one